layers/shared_classes_layer/python/data_service.py

```diff
@@ -69,3 +69,65 @@ def delete_marker(self, markerId):
         except Exception as e:
             logger.error(f"Error deleting marker with markerId {markerId}: {e}")
             return None
+
+    def update_marker(self, marker: LocationMarker) -> LocationMarker:
```

Comment:
@PCBush no need to return LocationMarker here. Let's make it not return anything. This function raises an error, so this is enough to know if write was successful or not

Reply:
Done